explainer.md

```diff
@@ -591,6 +591,90 @@ vrSession.addEventListener('resetpose', vrSessionEvent => {
 });
 ```
 
+### Presenting automatically when the user interacts with the headset
+
+Many VR devices have some way of detecting when the user has put the headset on or is otherwise trying to use the hardware. For example: an Oculus Rift or Vive have proximity sensors that indicate when the headset is being worn. And a Daydream device uses NFC tags to inform the phone when it's been placed in a headset. This is referred to as the `VRDevice` being "activated", and depending on context it may represent the user showing a clear intent to begin using VR. Many WebVR application may want to begin presenting automatically in these scenarios.
+
+In order to start presenting when the `VRDevice` is activated pages can request a deferred session. To request a deferred session the option `{ waitForActivate: true }` is passed into the `requestSession` call. When a session is requested with `waitForActivate` the request will remain outstanding until the `VRDevice` is activated, at which point the promise will resolve with the requested session (or reject, if necessary.)
+
+Deferred sessions must be exclusive and will not be fulfilled if there is already an exclusive session active for the VR hardware device when the activation action occurs. Deferred requests do not need to be made within a user gesture.
+
+```js
+// Requests that a session be created when the device is activated.
+vrDevice.requestSession({ exclusive: true, waitForActivate: true }).then(OnSessionStarted);
+```
+
+Once the deferred session request has been fulfilled or rejected a new deferred session request will need to be issued if the page wishes to respond to future activation actions.
+
+To detect when the user removes the headset, at which point the page may want to end the session, listen for the `deactivate` event. Note that not all devices capable of detecting activation can reliable detect deactivation, so pages are not guaranteed to receive an associated deactivate event for each activation action.
+
+```js
+vrDevice.addEventListener('deactivate', (vrDeviceEvent) => {
```

Comment:
Session can't be looked up from this event, because the device may have multiple sessions associated with it. I guess you could imply that we only care about the exclusive session in this case. In fact, you could even make this a session event (fired only on exclusive sessions),  which would then allow us to drop the `VRDeviceEvent` interface all together! Sounds appealing, any useful functionality that we would miss if this could only fire on sessions?

As for the `unloads`, I wouldn't expect this to fire in those cases. It should generally be a signal about the hardware state, not the page state.